src/conditions.lisp

```diff
@@ -231,6 +231,7 @@
 				                 :format-control "Assert failed: ~s."
 				                 :format-arguments (list form)))))
 
+#+nil
 (defmacro %%assert (test &optional ignore datum &rest args)
```

Comment:
Shoudl we then get rid of this definition and the commented out below?

Reply:
Yes. A couple of minutes.

Reply:
ready

@davazp, thanks for the review!